src/render/renderer.zig

```diff
@@ -2,12 +2,15 @@ const wlr = @import("../wlroots.zig");
 
 const wayland = @import("wayland");
 const wl = wayland.server.wl;
+const Texture = @import("texture.zig");
 
 pub const Renderer = extern struct {
     const Impl = opaque {};
+    const Egl = opaque {};
 
     impl: *const Impl,
 
+    format: *const Impl,
```

Comment:
why is this here?

Reply:
Apologies, testing code that got pushed by mistake.